### DISCUSSION TITLE
플레이 도중에 이름 표시 On / Off 하는 방법이 있나요?

### DISCUSSION THREAD

**Discussion by earthboom** [naverz/zepeto-studio-kor#983] [5. Q&A]
안녕하세요.

현재 플레이 도중에 본인, 타 유저의 닉네임을 On / Off를 하려고 합니다.

showNickNameLocalPlayer, showNickNameOtherPlayers 의 boolean 값을 런타임 중에 변경하였지만, 변경점이 보이지 않습니다.

플레이 도중에 컨트롤할 다른 방법이 있나요? 아니면 지원되지 않는 건가요?

**Comment by Zepeto-TechEvangelist** [comment]
네 현재는 불가합니다.
하지만 최근에 Floating UI 관련 동적 제어 니즈가 많아진 것을 감안하여
지원하는 방향으로 검토해 보겠습니다.
감사합니다.
